### ISSUE TITLE
KartAccelerationCharacter

### ISSUE THREAD

**Issue by zouzzz** [PeterLemon/N64#7]
Hello,
What are the differences in the lines if KartAccelerationCharacter ?

example :

KartAccelerationBowser:
            // ROM       RAM         Kart Acceleration Bowser
float32 2.0 // $000E3BE8 $800E2FE8 - Kart Acceleration Bowser (IEEE32) (2.0: $40000000)
float32 2.0 // $000E3BEC $800E2FEC - Kart Acceleration Bowser (IEEE32) (2.0: $40000000)
float32 2.0 // $000E3BF0 $800E2FF0 - Kart Acceleration Bowser (IEEE32) (2.0: $40000000)
float32 1.6 // $000E3BF4 $800E2FF4 - Kart Acceleration Bowser (IEEE32) (1.6: $3FCCCCCD)
float32 1.0 // $000E3BF8 $800E2FF8 - Kart Acceleration Bowser (IEEE32) (1.0: $3F800000)
float32 1.0 // $000E3BFC $800E2FFC - Kart Acceleration Bowser (IEEE32) (1.0: $3F800000)
float32 1.0 // $000E3C00 $800E3000 - Kart Acceleration Bowser (IEEE32) (1.0: $3F800000)
float32 1.8 // $000E3C04 $800E3004 - Kart Acceleration Bowser (IEEE32) (1.8: $3FE66666)
float32 1.8 // $000E3C08 $800E3008 - Kart Acceleration Bowser (IEEE32) (1.8: $3FE66666)
float32 1.2 // $000E3C0C $800E300C - Kart Acceleration Bowser (IEEE32) (1.2: $3F99999A)

Thanks.

**Comment by zouzzz** [comment]
Sorry,

"What are the differences in the lines OF KartAccelerationCharacter ?"

**Comment by PeterLemon** [comment]
Hi zouzzz,

There is no difference in those Kart Acceleration numbers.

When I made this hack, I only changed a few numbers:
KartSpeed50cc
KartSpeed100cc
KartSpeed150cc
KartSpeedExtra
KartSpeedBattle

I only added all the other numbers I could find like Kart Acceleration, so people can try changing those settings themselves.
The reason the karts accelerate so much faster than normal, is that increasing the Kart Speed numbers also increases the acceleration.

Hope this helps =D
